### PR TITLE
Set a font-size for pre elements

### LINKED ISSUE #412: Code blocks in docs have different font sizes
HTML snippets: 11.6667px

image

JS snippets: 13.3333px

image

### PR DESCRIPTION
**Pull request by leebyp**
#### Changes proposed in this pull request:

Set a font-size for `<pre />` elements.

### CODE REVIEW

**Comment by llorca** [comment]
A bit concerned that 13px comes out of nowhere and that it doesn't depend on any other variable, but the sizes does look good. We don't use rems, maybe we can do some math around `$pt-font-size`?

File: packages/core/src/_typography.scss [hunk 1]
```diff
@@ -299,6 +299,7 @@ pre {
   padding: ($pt-grid-size * 1.3) ($pt-grid-size * 1.5) ($pt-grid-size * 1.2);
   line-height: 1.4;
   color: $pt-text-color;
+  font-size: $pt-grid-size * 1.3;
```
Comment (giladgray) [inline packages/core/src/_typography.scss]: how about `$pt-font-size - 1px`?

**Comment by leebyp** [comment]
Looking through our variables, I feel like it looks decent because it's the middle between `@pt-font-size: 14px` and `@pt-font-size: 13px` rather than being math from anything in particular. We could do `$pt-font-size - 1px` which encapsulates part of this I guess?

File: packages/core/src/_typography.scss (hunk 1 above)
Comment (giladgray) [inline packages/core/src/_typography.scss]: this is like a hardcoded `smaller` so it makes sense to derive from original font size.

Comment (leebyp) [inline packages/core/src/_typography.scss]: oh snap!

**Pull request merged**